google/auth/_helpers.py

```diff
@@ -162,10 +162,10 @@ def update_query(url, params, remove=None):
     # Update the query parameters with the new parameters.
     query_params.update(params)
     # Remove any values specified in remove.
-    query_params = {
+    query_params = dict({
```

Comment:
Empirically this is false.

```python
>>> vals = {a: a * a for a in [1, 2, 3]}
>>> vals
{1: 1, 2: 4, 3: 9}
>>> type(vals)
<class 'dict'>
>>>
```

and I got identical output in 2.7, 3.4, 3.5 and 3.6